openquake/calculators/event_based_risk.py

```diff
@@ -299,13 +297,12 @@ def _save_maps(self, dic, aids):
                 for aid, arr in zip(aids, loss_maps):
                     self.datastore[key][aid] = arr
 
-    def combine(self, dummy, results):
+    def combine(self, dummy, res):
         """
         :param dummy: unused parameter
         :param res: a list of result dictionaries
```

Comment:
If this is now a single item instead of a list, we should also update the docstring accordingly